pkg/restapi/vc/operation/operations.go

```diff
@@ -775,20 +779,88 @@ func (o *Operation) GetRESTHandlers(mode string) ([]Handler, error) {
 		}, nil
 	case "issuer":
 		return []Handler{
-			support.NewHTTPHandler(createCredentialEndpoint, http.MethodPost, o.createCredentialHandler),
+			// profile
 			support.NewHTTPHandler(createProfileEndpoint, http.MethodPost, o.createProfileHandler),
 			support.NewHTTPHandler(getProfileEndpoint, http.MethodGet, o.getProfileHandler),
+
+			// verifiable credential
+			support.NewHTTPHandler(createCredentialEndpoint, http.MethodPost, o.createCredentialHandler),
 			support.NewHTTPHandler(storeCredentialEndpoint, http.MethodPost, o.storeVCHandler),
 			support.NewHTTPHandler(verifyCredentialEndpoint, http.MethodPost, o.verifyCredentialHandler),
 			support.NewHTTPHandler(updateCredentialStatusEndpoint, http.MethodPost, o.updateCredentialStatusHandler),
 			support.NewHTTPHandler(retrieveCredentialEndpoint, http.MethodGet, o.retrieveVCHandler),
 			support.NewHTTPHandler(vcStatusEndpoint, http.MethodGet, o.vcStatus),
+
+			// issuer apis
+			// TODO update trustbloc components to use these APIs instead of above ones
+			support.NewHTTPHandler(issueCredentialPath, http.MethodPost, o.issueCredentialHandler),
 		}, nil
 	default:
 		return nil, fmt.Errorf("invalid operation mode: %s", mode)
 	}
 }
 
+func (o *Operation) issueCredentialHandler(rw http.ResponseWriter, req *http.Request) {
```

Comment:
we need bdd-test

Reply:
I'll add once https://github.com/trustbloc/edge-service/issues/123 is done (all components for integration available).

Reply:
Issue for BDD tests - https://github.com/trustbloc/edge-service/issues/124